src/apps/user-management/InviteUserDialog/InviteUserDialog.css

```diff
@@ -0,0 +1,134 @@
+.invite-user-dialog-overlay {
```

Comment:
There's actually no need for `.invite-user-dialog-overlay`. If you just set the class to `.dialog-overlay`, it will pick up the default styles from the main stylesheet.